Root/TruthAnalysis.cxx

```diff
@@ -626,6 +639,11 @@ EL::StatusCode TruthAnalysis :: finalize ()
   // submission node after all your histogram outputs have been
   // merged.  This is different from histFinalize() in that it only
   // gets called on worker nodes that processed input events.
+
+  // output xAOD
+  TFile *file_xAOD = wk()->getOutputFile ("output_xAOD");
+  EL_RETURN_CHECK("finalize()", m_event->finishWritingTo( file_xAOD ));//, "Could not finish writing to the output xAOD");
+
```

Comment:
Probably just want an output nTuple, or histograms, instead of xAOD. We don’t need anything fancy and reading it back will be much easier!

On Oct 29, 2015, at 12:28 PM, Giordon Stark <notifications@github.com<mailto:notifications@github.com>> wrote:


In Root/TruthAnalysis.cxx<https://github.com/mattleblanc/MultibjetsTruthAnalysis/pull/1#discussion_r43417165>:

> @@ -626,6 +639,11 @@ EL::StatusCode TruthAnalysis :: finalize ()
>    // submission node after all your histogram outputs have been
>    // merged.  This is different from histFinalize() in that it only
>    // gets called on worker nodes that processed input events.
> +
> +  // output xAOD
> +  TFile *file_xAOD = wk()->getOutputFile ("output_xAOD");
> +  EL_RETURN_CHECK("finalize()", m_event->finishWritingTo( file_xAOD ));//, "Could not finish writing to the output xAOD");
> +


Make sure we close up the pointers and shit.

—
Reply to this email directly or view it on GitHub<https://github.com/mattleblanc/MultibjetsTruthAnalysis/pull/1/files#r43417165>.